docs/cloud/pouta/tutorials/image-pipeline.md

```diff
@@ -0,0 +1,650 @@
+| name              | None                                                                                                                                                                       |
+| port_range_max    | 22                                                                                                                                                                         |
+| port_range_min    | 22                                                                                                                                                                         |
+| project_id        | ef20bcc4215d49ddbf33e5c5740d89fe                                                                                                                                           |
+| protocol          | tcp                                                                                                                                                                        |
+| remote_group_id   | None                                                                                                                                                                       |
+| remote_ip_prefix  | 0.0.0.0/0                                                                                                                                                                  |
+| revision_number   | 0                                                                                                                                                                          |
+| security_group_id | a8630776-db3d-408a-ba8e-8c52b5f2a8c9                                                                                                                                       |
+| tags              | []                                                                                                                                                                         |
+| updated_at        | 2025-02-04T13:13:04Z                                                                                                                                                       |
++-------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+```
+
+Now we apply the security group to the previously created virtual machine, so that this newly-created rule applies to its traffic.
+```
+$ openstack server add security group pipeline_vm pipeline_security_group
+```
+In case of success, the command will show no output.
+
+### Connecting to the virtual machine
+
+The virtual machine is now configured to allow traffic from our workstation but it is not reachable yet because it has no address assigned to it.
```

Comment:
```suggestion
The virtual machine is now configured to allow traffic from our workstation but it is not reachable yet because it has no public IP address assigned to it.
```

It should be clarified that VM need a public ip and that VMs only get private IPs by default.

Reply:
Fixed in https://github.com/CSCfi/csc-user-guide/pull/2424/commits/024dc7859b0230ca1fbff7675d9101169380862c